jenkins/check_copyright.sh

```diff
@@ -37,6 +37,8 @@ git diff --name-only `git merge-base origin/$ghprbTargetBranch HEAD` HEAD |
     grep -v -E '\.cmake$' |
     grep -v -E '\.json$' |
     grep -v -E '\.man$' |
+    grep -v -E '\.proj$' |
```

Comment:
BTW, why is this new project file a .proj instead of .vcxproj?

Reply:
I wasn't sure if `vcxproj` had connotations of being a particular kind of object. The example project that the new file is based on was also just a `.proj` file.